Vagrantfile

```diff
@@ -0,0 +1,78 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile is used for BPF testing. See also bpf/README.md.
+#
+# To build Teleport, try to build as much as you can outside of Vagrant and
```

Comment:
The experience here isn't super slick, but it's a load better than nothing and it does capture relevant information, like kernel versions, required dev libraries, where to place libbpf, etc.